### ISSUE TITLE
table wrap issue

### ISSUE THREAD

**Issue by jamiepreston** [gavinelliott/wfms#1]
If the content in the table on the caravans form page (`.data-table`) is extended beyond a certain point, the whole `admin-content` pane drops down below the nav bar.  This is happening with some of our test data.

**Comment by gavinelliott** [comment]
@jamiepreston Can you share a screenshot?

**Comment by jamiepreston** [comment]
Here's the form before wrapping:
![prototype1](https://cloud.githubusercontent.com/assets/20536110/18084708/1ec4b7b4-6ea0-11e6-9c7a-69c5fb08abe5.PNG)

If the viewport gets any smaller this happens on the prototype:
![prototype2](https://cloud.githubusercontent.com/assets/20536110/18084709/1eca0192-6ea0-11e6-873e-6425fc2d4eeb.PNG)

Then the form comes back in at the smaller viewports:
![prototype3](https://cloud.githubusercontent.com/assets/20536110/18084737/45ffd25a-6ea0-11e6-95ba-99d2519613dd.PNG)

Thanks

**Comment by gavinelliott** [comment]
@jamiepreston I love a challenge.

**Comment by gavinelliott** [comment]
@jamiepreston I've just pushed a fix which should work.

**Comment by jamiepreston** [comment]
Thanks @gavinelliott but I can't see any recent commits - did you push?

**Comment by gavinelliott** [comment]
I did, but it had a fit over my not being auth'd. Sorted now and should be there for you.

**Comment by jamiepreston** [comment]
That hasn't sorted it unfortunately mate.  :-(

I'll explain in a bit more detail as to why it's such a big issue in the WFMS app we're developing.  Unlike the prototype, we have a button which moves the user on to the next set of records.  Obviously the data is different on the next page, therefore it's making the table a different width.  On one set of data, the page layout is neatly sitting to the right of the menu, but then pressing the next button makes the whole content div jump down underneath the menu.  

Here's a couple of screenshots using the prototype so you can see the effect a longer `<td>` content has:

![prototype1](https://cloud.githubusercontent.com/assets/20536110/18135741/d3008f3c-6f9a-11e6-8ad4-0d2422258eb6.PNG)

Without resizing the window, I've used chrome's developer tools to add a longer DOM (this is the date format provided to us at the minute):

![prototype2](https://cloud.githubusercontent.com/assets/20536110/18135767/e5376bb2-6f9a-11e6-9564-a7027c4c9213.PNG)

The viewport size I have there is 1380px, using a different browser doesn't seem to make a difference.

**Comment by gavinelliott** [comment]
No worries, leave it with me. Will have something back across today, it might mean a change to the nav, but will see how I go.

**Comment by jamiepreston** [comment]
ok great, thanks for looking into it.

**Comment by jamiepreston** [comment]
How's this looking @gavinelliott, any further forward?

**Comment by gavinelliott** [comment]
Working on it right now, @jamiepreston, sorry for the delay.

**Comment by gavinelliott** [comment]
@jamiepreston I've just pushed up some changes which are quite drastic. Give it a huge refresh once you hit the link. I've changed the navigation and am still working on some styling and layout issues. 

Overall, you'll notice that if you go to the link > Forms > Sub-menu of WFP Caravans, the table is now full width within the window. Let me know once you've pushed some data in there how it sits. I'll be popping across to your place tomorrow to see Mel so I can chat with you then.
